crates/example-types/src/storage_types.rs

```diff
@@ -64,4 +63,19 @@ impl<TYPES: NodeType> Storage<TYPES> for TestStorage<TYPES> {
             .insert(proposal.data.view_number, proposal.clone());
         Ok(())
     }
+
+    async fn record_action(
+        &self,
+        _view: <TYPES as NodeType>::Time,
+        _action: hotshot_types::event::HotShotAction,
+    ) -> Result<()> {
+        Ok(())
+    }
+
+    async fn update_high_qc(
+        &self,
+        _qc: hotshot_types::simple_certificate::QuorumCertificate<TYPES>,
+    ) -> Result<()> {
+        Ok(())
+    }
```

Comment:
Should these have some sort of dummy implementation?

Reply:
They could, but there isn't a way to even read from this so I'm not sure how valuable that would be.